### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by endJunction**
some of the warnings shown by msvc.

### CODE REVIEW

**Comment by TomFischer** [comment]
Looks good. :+1:

File: MeshLib/convertMeshToGeo.cpp [hunk 1]
```diff
@@ -63,12 +63,11 @@ bool convertMeshToGeo(const MeshLib::Mesh &mesh, GeoLib::GEOObjects &geo_objects
...
     const std::size_t nElems (mesh.getNumberOfElements());
 
-    auto materialIds = mesh.getProperties().getPropertyVector<int>(mat_name);
-    auto const materialIdExist = bool(materialIds);
+    auto const materialIds = mesh.getProperties().getPropertyVector<int>(mat_name);
 
     for (unsigned i=0; i<nElems; ++i)
     {
-        auto surfaceId = materialIdExist ? (*materialIds)[i]-bounds.first : 0;
+        auto surfaceId = !materialIds ? 0 : (*materialIds)[i] - bounds.first;
```
Comment (norihiro-w) [inline MeshLib/convertMeshToGeo.cpp]: i'm unsure if the line works in the same way as `auto surfaceId = !materialIds ? 0 : ((*materialIds)[i] - bounds.first);`

**Comment by norihiro-w** [comment]
looks good

File: MeshLib/convertMeshToGeo.cpp (hunk 1 above)
Comment (endJunction) [inline MeshLib/convertMeshToGeo.cpp]: Assembly is equal. But I'm adding the parenthesis anyway for clarity.

**Pull request merged**